scripts/question-parser.ts

```
@@ -0,0 +1,148 @@
+      const lines = content.split('\n');
+      
+      let currentCategory = '';
+      let currentSection = '';
+      let questionId = 1;
+      
+      for (const line of lines) {
+        const trimmedLine = line.trim();
+        
+        // Detect category headers (### 🏥 Salud)
+        if (trimmedLine.startsWith('### ') && trimmedLine.includes(' ')) {
+          const categoryMatch = trimmedLine.match(/### [🔥🧓🏥🎓💼💰🚨🏠🌱🌍🏛️]*\s*(.+)/);
```

Biome (2.1.2)

[error] A character class cannot match a surrogate pair. Add the 'u' unicode flag to match against them.

A surrogate pair forms a single codepoint, but is encoded as a pair of two characters. Without the unicode flag, the regex matches a single surrogate character.
Safe fix: Add unicode u flag to regex


(lint/suspicious/noMisleadingCharacterClass)